### DISCUSSION TITLE
Redirect back to the actual search result page instead of the first page in search results webpart

### DISCUSSION THREAD

**Discussion by Praveen-S2** [microsoft-search/pnp-modern-search#1950] [Ideas]
### Check existing issues before logging new ones
If you are not using the latest version, triage your issue with the latest version before logging as we won't provide patches to older versions.

### Issues related to data mapping, e.g. SharePoint columns to crawled properties to managed properties should be asked in the discussion section.

**For any question regarding Web Parts usage, please use the ["Discussions"](https://github.com/microsoft-search/pnp-modern-search/discussions) tab instead of creating a new issue.**.

__Remove the above section before submitting, and removal of the below template will result in the issue being closed.__

**Version used**
v4

**Describe the bug**
When I click on the search result from second page in the search results webpart and then if I click on the back button in the browser, it will redirect me to the 1st page of the search results page instead of the second page.

**Comment by FranckyC** [comment]
Hi @Praveen-S2, this is the expected behavior as we do not use window history in the pagination control. Move it into discussion as enhancement.
